CocoaLumberjack.podspec

```diff
@@ -2,7 +2,7 @@
 Pod::Spec.new do |s|
 
   s.name     = 'CocoaLumberjack'
-  s.version  = '2.3.0'
+  s.version  = '2.4.0'
```

Comment:
Swift 3.0 has a breaking API changes. I'll suggest to use `3.0.0` version instead.